styles/about_me.css

```diff
@@ -0,0 +1,71 @@
+.gray__bg {
+  position: absolute;
+  width: 70vw;
+  height: 420px;
+  left: 0px;
+  background-color: #fafaff;
+}
```

Comment:
Wydaje mi się, że tutaj najlepszym pomysłem by było zrobić div który opakowuje wszystkie elementy w sekcji i z niego zrobić position absolute, wtedy wysokość będzie z automatu taka wysoka jak sekcja, ew. dodać wtedy margines jakiś, i chyba jeżeli chcemy te tła mieć to jedynym pomysłem jest właśnie pozycja absolutna i przesunięcie